pkg/query/args.go

```diff
@@ -287,13 +287,10 @@ func parseIfaceList(dbPath string, ifacelist string) (ifaces []string, err error
 	}
 
 	if strings.ToLower(ifacelist) == "any" {
-		summary, err := goDB.ReadDBSummary(dbPath)
+		ifaces, err = goDB.GetInterfaces(dbPath)
```

Comment:
At last, the `any` query is fixed, even if the `summary.json` is broken 😌

Reply:
:man_shrugging: